ush/python/pygfs/task/atm_analysis.py

```diff
@@ -0,0 +1,436 @@
+
+        # Start accumulating list of background files to copy
+        bkglist = []
+
+        # atm DA needs coupler
+        basename = f'{to_fv3time(task_config.current_cycle)}.coupler.res'
+        bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        # atm DA needs core, srf_wnd, tracer, phy_data, sfc_data
+        for ftype in ['core', 'srf_wnd', 'tracer']:
+            template = f'{to_fv3time(self.task_config.current_cycle)}.fv_{ftype}.res.tile{{tilenum}}.nc'
+            for itile in range(1, task_config.ntiles + 1):
+                basename = template.format(tilenum=itile)
+                bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        for ftype in ['phy_data', 'sfc_data']:
+            template = f'{to_fv3time(self.task_config.current_cycle)}.{ftype}.tile{{tilenum}}.nc'
+            for itile in range(1, task_config.ntiles + 1):
+                basename = template.format(tilenum=itile)
+                bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        bkg_dict = {
+            'mkdir': [run_dir],
+            'copy': bkglist,
+        }
+        return bkg_dict
+
+    @logit(logger)
+    def get_berror_dict(self, config: Dict[str, Any]) -> Dict[str, List[str]]:
+        """Compile a dictionary of background error files to copy
+
+        This method will construct a dictionary of either bump of gsibec background
+        error files for global atm DA and return said dictionary for use by the
+        FileHandler class.
+
+        Parameters
+        ----------
+        config: Dict
+            a dictionary containing all of the configuration needed
+
+        Returns
+        ----------
+        berror_dict: Dict
+            a dictionary containing the list of atm background error files to copy for FileHandler
+        """
+        SUPPORTED_BERROR_STATIC_MAP = {'identity': self._get_berror_dict_identity,
+                                       'bump': self._get_berror_dict_bump,
+                                       'gsibec': self._get_berror_dict_gsibec}
+
+        try:
+            berror_dict = SUPPORTED_BERROR_STATIC_MAP[config.STATICB_TYPE](config)
+        except KeyError:
+            raise KeyError(f"{config.STATICB_TYPE} is not a supported background error type.\n" +
+                           f"Currently supported background error types are:\n" +
+                           f'{" | ".join(SUPPORTED_BERROR_STATIC_MAP.keys())}')
+
+        return berror_dict
+
+    @staticmethod
+    @logit(logger)
+    def _get_berror_dict_identity(config: Dict[str, Any]) -> Dict[str, List[str]]:
+        """Compile a dictionary of atm identity background error files to copy
+        This method is not implemented yet.
+
+        This is a private method and should not be accessed directly.
+        """
+        raise NotImplementedError("Identity background error not yet implemented")
```

Comment:
This isn't quite correct, there are no files to copy, so it is 'implemented' as long as the YAML template is there.

Reply:
Good point @CoryMartin-NOAA .  We can rewrite this as

```
         This is a private method and should not be accessed directly.
         """
-        raise NotImplementedError("Identity background error not yet implemented")
+        logger.info(f"Identity background error does not stage files. Return with no action.")

     @staticmethod
     @logit(logger)
```

Is this better?

Reply:
Something like that yes would be great

Reply:
Change committed at [1ca8a2a](https://github.com/RussTreadon-NOAA/global-workflow/commit/1ca8a2a431c884ba5f17ce9ef876c63ab1cc5ea8).   Both comment and runtime message updated.